dcicutils/deployment_utils.py

```diff
@@ -319,15 +339,15 @@ def terminate_indexer_env(client, env_name):
             NOTE: there is 1 hr timeout before you can recreate one of these for the same application.
 
         :param client: boto3 elasticbeanstalk client
-        :param env_name:
+        :param env_name: one of: FF_ENV_INDEXER or CGAP_ENV_INDEXER
```

Comment:
Hmm. The idea of the function is not to use these variables at all. But it's a small matter so I'd leave it.